### ISSUE TITLE
On*LoadDocumentObject accept the result only from the first plugin

### ISSUE THREAD

**Issue by mnoskov** [evolution-cms/evolution#1579]
Events OnBeforeLoadDocumentObject, OnLoadDocumentObject and OnAfterLoadDocumentObject accept the result only from the first plugin, ignoring next ones.

For example, if we have two plugins on event OnLoadDocumentObject, and both set documentObject, only first will be accepted.

**Comment by Dmi3yy** [comment]
А это не оно ?

http://modx.im/blog/docs/5933.html

**Comment by mnoskov** [comment]
Нет, getOutput/setOutput работают в рамках обработки одного плагина. А проблема в том, что на событии висит несколько плагинов.

**Comment by Dmi3yy** [comment]
Если не ошибаюсь то событие в админке с виджетами работает коректно но там в коде идет обработка того что б оно обрабатывалось поочередно. 
Нужно там глянуть и если соо добавить во все нужные места по аналогии
